components/Header/style.js

```diff
@@ -1,13 +1,14 @@
 import {StyleSheet, Dimensions} from 'react-native';
 
-const windowWidth = Dimensions.get('window').width;
-const windowHeight = Dimensions.get('window').height;
+// const windowWidth = Dimensions.get('window').width;
+// const windowHeight = Dimensions.get('window').height;
 
 const styles = StyleSheet.create({
   headerContainer: {
-    width: windowWidth * 0.9,
-    height: windowHeight * 0.05,
-    marginTop: windowHeight * 0.04,
+    // width: windowWidth,
+    // width: windowWidth,
+    // height: windowHeight * 0.1,
+    marginTop: 32,
```

Comment:
why did you comment these codes?

Reply:
It expected to use late, but removed for now.